forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -44,12 +45,14 @@ def get_mapper_count(form_name=None):
         return FormProcessMapper.find_all_count()
 
     @staticmethod
-    def get_mapper(form_process_mapper_id: int):
+    def get_mapper(form_process_mapper_id: int, tenant_key: Any = None):
```

Comment:
change the tenantKey to str (not Any) as key is always string ?

Reply:
NoneType in case of no tenant key is present, could use Optional[str] = None